semgrep_metrics.atd

```diff
@@ -129,9 +133,16 @@ type extension = {
 (* Misc *)
 (*****************************************************************************)
 
+type pro_features = {
+    (* The value is determined by the `--diff-depth` option, and it's utilized
+       for inter-file differential scanning. Since semgrep 1.46 *)
+    ?diffDepth <ocaml mutable>: int option;
+}
+
 type misc = {
     (* coupling: features is commented a lot in semgrep/PRIVACY.md *)
     features <ocaml mutable>: string list;
+    proFeatures <ocaml mutable>: pro_features;
```

Comment:
this need to be ?proFeatures ... : pro_features option;
because the metrics backend (which in theory should use this interface file too) need to remain backward compatible and be able to accept metrics generated by old version of Semgrep. See the toplevel comment about protocol update in semgrep_output_v1.atd

Reply:
Updated.